openslide/deepzoom.py

```diff
@@ -97,8 +97,11 @@ def __init__(self, osr, tile_size=254, overlap=1, limit_bounds=False):
             z_size = tuple(max(1, int(math.ceil(z / 2))) for z in z_size)
             z_dimensions.append(z_size)
         self._z_dimensions = tuple(reversed(z_dimensions))
+
         # Tile
-        tiles = lambda z_lim: int(math.ceil(z_lim / self._z_t_downsample))
+        def tiles(z_lim, _z_t_downsample=self._z_t_downsample):
```

Comment:
Leading underscore in name of argument which is, in fact, used.  But also, does this need to be an argument?  The callers don't pass it.

Reply:
I'm never quite confident I got scoping correct, so it seemed safer to pass the variable as the default value of an argument. But yes, the underscore is misleading because the argument is in fact used by the function.